telemetry/src/config.rs

```diff
@@ -16,8 +16,22 @@ pub struct Configuration {
     /// The url of the telemetry, e.g., ws://127.0.0.1:8001/submit
     #[config(serde_as_str)]
     pub url: Option<Url>,
+    /// The minimum period of time in seconds to wait before reconnecting
+    #[serde(default = "default_min_period")]
+    pub min_period: u64,
+    /// The maximum power of 2 that is used for increasing delay between reconnections
+    #[serde(default = "default_max_power")]
+    pub max_power: u64,
```

Comment:
```suggestion
    pub max_power_exponent: u8,
```
1) I interpreted `max_power` as being 2^`max_power` not just `max_power`. The correct term here is exponent. 

2) I'm quite sure that a retry period of 2^128 cannot be machine rerpresentable as even `u128` which is the maximum integer we use internally, so `u64` is not only overkill, but communicates the wrong idea.

Reply:
@eadventurous Do we want to have some the maximum of max_power?

Reply:
I agree with @appetrosyan here that `u8` would make more sense. You can also do a [checked_pow](https://doc.rust-lang.org/nightly/std/primitive.u64.html#method.checked_pow) to ensure that we don't overflow.